modules/oracle-free/src/main/java/org/testcontainers/oracle/OracleContainer.java

```diff
@@ -0,0 +1,194 @@
+package org.testcontainers.oracle;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Testcontainers implementation for Oracle Database Free.
+ * <p>
+ * Supported image: {@code gvenzl/oracle-free}
+ * <p>
+ * Exposed ports: 1521
+ */
+public class OracleContainer extends JdbcDatabaseContainer<OracleContainer> {
+
+    static final String NAME = "oracle";
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("gvenzl/oracle-free");
+
+    static final String DEFAULT_TAG = "slim";
+
+    static final String IMAGE = DEFAULT_IMAGE_NAME.getUnversionedPart();
+
+    static final int ORACLE_PORT = 1521;
+
+    private static final int DEFAULT_STARTUP_TIMEOUT_SECONDS = 60;
+
+    private static final int DEFAULT_CONNECT_TIMEOUT_SECONDS = 60;
+
+    // Container defaults
+    static final String DEFAULT_DATABASE_NAME = "freepdb1";
+
+    static final String DEFAULT_SID = "free";
+
+    static final String DEFAULT_SYSTEM_USER = "system";
+
+    static final String DEFAULT_SYS_USER = "sys";
+
+    // Test container defaults
+    static final String APP_USER = "test";
+
+    static final String APP_USER_PASSWORD = "test";
```

Comment:
We really should default to secure random passwords in the future (in the baseclass)